languages/es.json

```
@@ -68,20 +68,20 @@
     "Send Us Feedback": "Envianos tus comentarios",
     "Sign up for accessibility access to bypass the hCaptcha challenge.": "Regístrese para acceder a la accesibilidad para evitar el desafío hCaptcha.",
     "Skip": "Omitir",
-    "Skip Challenge": "Saltar desafío",
+    "Skip Challenge": "Omitir prueba.",
     "Software Bug": "Error de programación",
     "Sorry to hear that!": "Lamento escuchar eso!",
     "Status:": "Estado:",
     "Submit Answers": "Presentar respuestas",
     "Tell Us Why": "Dinos por qué",
     "Terms": "Condiciones",
-    "Thank you for your feedback.": "Gracias por tus comentarios.",
+    "Thank you for your feedback.": "Gracias por sus comentarios.",
     "The sitekey for this hCaptcha is incorrect. Please contact the site admin if you see this.": "La clave de sitio para este hCaptcha es incorrecta. Póngase en contacto con el administrador del sitio si ve esto.",
     "This hCaptcha is for testing only. Please contact the site admin if you see this.": "Este hCaptcha es solo para pruebas. Póngase en contacto con el administrador del sitio si ve esto.",
     "This lets you avoid future questions by registering and setting a cookie.": "Esto le permite evitar futuras preguntas al registrarse y configurar una cookie.",
-    "To bypass our visual challenge, we offer an accessibility cookie.": "Para evitar nuestro desafío visual, ofrecemos una cookie de accesibilidad.",
+    "To bypass our visual challenge, we offer an accessibility cookie.": "Para evitar nuestra prueba visual, ofrecemos una cookie de accesibilidad.",
     "Too Difficult": "Demasiado difícil",
-    "View more examples.": "Ver más ejemplos.",
+    "View more examples.": "Ver menos ejemplos.",
```

```suggestion
    "View more examples.": "Ver más ejemplos.",
```

This seems erroneous.